PyOpenWorm/evidence.py

```diff
@@ -84,5 +84,36 @@ def identifier_augment(self):
         s += self.reference.defined_values[0].identifier.n3()
         return self.make_identifier(s)
 
-
+def evidence_for(qctx, conn, Evidence, Context):
```

Comment:
I know why you're including the connection here, but I think it should be a `Context` instead so I can restrict where the `Evidence::supports` statements are retrieved from.

On the other hand, I was thinking about it -- it may be a better to do something like `Evidence.for(obj)`. `Evidence` is already a `ContextualizableClass`, so we can get the class' context from the target of `for`. Ex:
```
qctx = Context()
# make statements in qctx
qqctx(Evidence).for(qctx)
```
what do you think?

Reply:
#209 described a evidenced_by() which seemed simple and straight forward, ill look into the implementation using the example described above!